### PR TITLE
Updated alert_handling_job

### PR DESCRIPTION
**Pull request by pavangudiwada**
* This action can now create a Job that accept secrets, wait for a job to complete etc. 
* Secret injection into the job is working as expected

### CODE REVIEW

File: docs/tutorials/alert-remediation.rst [hunk 1]
```diff
@@ -13,23 +13,40 @@ Add the following to your :ref:`customPlaybooks<customPlaybooks>`:
...
-          command:
-          - "echo"
-          - "do something to fix it"
-          image: "a_docker_image"
+        - alert_handling_job:
+            # you can access information from the alert by environment variables
+            command:
+              - sh
+              - -c
+              - "echo '$ALERT_NAME fired... Now dumping all available environment variables, which include alert metadata and labels' && env && sleep 60"
```
Comment (aantn) [inline docs/tutorials/alert-remediation.rst:26]: @pavangudiwada can you add to the example something explicit showing how to print one of the labels from the alert?

File: docs/tutorials/alert-remediation.rst [hunk 2]
```diff
@@ -44,7 +62,7 @@ Alerts can also be remediated with bash commands:
           alert_name: SomeAlert
       actions:
       - node_bash_enricher:
-         bash_command: do something
+        bash_command: do something
```
Comment (aantn) [inline docs/tutorials/alert-remediation.rst:65]: @pavangudiwada did we remove necessary whitespace here?

Comment (pavangudiwada) [reply docs/tutorials/alert-remediation.rst:65]: Yes, fixed it thanks.

**Review by aantn — APPROVED** [review]
LGTM. Please merge when you're done testing. Nice work.

**Pull request merged**